docs/readthedocs/io/csvfiles.md

```diff
@@ -144,6 +144,9 @@ The following keys are supported until now:
    * - pqh
      - Active, reactive and heat power
        Permissible head line: ``time,p,q,h``
+   * - v
+     - Voltage mangnitude in pu and angle in °
+       Permissible head line: ``time,vMag,vAng``
```

Comment:
The line break does not work here. Maybe just add a `.` and keep it in the same line? Also, it'd be great if you could fix the other rows while you're at it. Also, table could use `:class: wrapping`